charts/mastodon/README.md

```diff
@@ -100,4 +100,23 @@ postgresql:
 
 And make sure to set `password` to the same value as `postgres-password`
 in your `mastodon-postgresql` secret:
-```kubectl edit secret mastodon-postgresql```
\ No newline at end of file
+```kubectl edit secret mastodon-postgresql```
+
+# Contributing
+
+We're happy to have changes from the community to help build and improve the Mastodon helm chart.
+All changes should be accompanied by an update to the `charts/mastodon/CHANGELOG.md` file.
+
+## Versioning
+
+When a contributor is ready to deploy a new version of the helm chart, please take the following steps:
+
+1. Update the `charts/mastodon/CHANGELOG.md` file with a new header section with the release number and today's date
+  - Such as `[2.4.0] - 2022-11-14`
+1. Update the version number in `charts/mastodon/Chart.yaml`
+
+After the PR is merged into main, a new release will be created in Github and the chart will be deployed to Github pages.
```

Comment:
I'm concerned about this strategy from a long term open-source maintenance perspective. I think its typically adding a lot of additional work for maintainers and contributors. There will be plenty of merge conflicts between version bumps etc for example.

At the same time, I don't have a great strategy to handle this if this Helm chart remains bundled with the mastodon/mastodon repo, and can't get its own version control.

I'm inclined to suggest that we don't introduce a changelog in this PR, which is by itself a bit of a stretch of scope.

Reply:
That works, I think it's worth discussing the changelog outside of this PR or perhaps another approach. Happy to make incremental progress :+1: